init-odkx-sync-endpoint.py

```diff
@@ -148,16 +164,18 @@ def run_docker_builds():
 
 
 def run_sync_endpoint_build():
-    os.system("git clone -b master --single-branch --depth=1 https://github.com/odk-x/sync-endpoint ; \
```

Comment:
why would we want to clone more than the single-branch for the automated build? Takes up more space and takes longer